### PR TITLE
Convert hui-picture-entity-card to TypeScript/LitElement

### PR DESCRIPTION
**Pull request by iantrich**
click/hold are not working on my Chrome dev env as well as other cards. Need to investigate this further before merging.

also need to create docs for the additional action option `call-service`

### CODE REVIEW

File: src/panels/lovelace/cards/hui-picture-entity-card.ts [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+        }
+        .state {
+          text-align: right;
+        }
+      </style>
+    `;
+  }
+
+  private handleClick(hold: boolean): void {
+    const stateObj = this.hass!.states[this._config!.entity];
```
Comment (balloob) [inline src/panels/lovelace/cards/hui-picture-entity-card.ts]: You should start the function with this:

```ts
const hass = this.hass!
const config = this._config!
```

Then you can get type checking back and drop all other `!`

File: src/panels/lovelace/cards/hui-picture-entity-card.ts [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+      footer = html`
+        <div class="footer state">${state}</div>
+      `;
+    }
+
+    return html`
+      ${this.renderStyle()}
+      <ha-card
+        id="card"
+        @ha-click="${() => this.handleClick(false)}"
```
[the remark this answers is not in the document]
Comment (iantrich) [reply src/panels/lovelace/cards/hui-picture-entity-card.ts]: Doh. I knew better. Thanks

File: src/panels/lovelace/cards/hui-picture-entity-card.ts [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+    const stateObj = this.hass!.states[this._config.entity];
+
+    if (!stateObj) {
+      throw new Error("State Ojbect does not exist for this entity");
+    }
+
+    const available = !(stateObj.state === UNAVAILABLE);
+
+    let footer = html``;
```
[the remark this answers is not in the document]
Comment (iantrich) [reply src/panels/lovelace/cards/hui-picture-entity-card.ts]: You can't assign a `TemplateResult` to a `string`

[the remark this answers is not in the document]
Comment (balloob) [reply src/panels/lovelace/cards/hui-picture-entity-card.ts]:
```suggestion
    let footer: TemplateResult | string = "";
```

**Review by balloob — APPROVED** [review]

**Pull request merged**